packages/orama/src/trees/bkd.ts

```diff
@@ -0,0 +1,356 @@
+import type { Nullable, GenericSorting } from '../types.js'
+import type { InternalDocumentID } from '../components/internal-document-id-store.js'
+
+export interface Point {
+  lon: number
+  lat: number
+}
```

Comment:
I think is more efficient and fast to store the data as an array (tuple).

Reply:
Taking the courage to refactor now :)